    Blocking waiting for file lock on build directory
    Checking google-dataproc1_beta2 v0.1.0 (/Users/sthiel/dev/google-apis/generated/gen/dataproc/v1beta2/lib)
error[E0433]: failed to resolve: unresolved import
     --> gen/dataproc/v1beta2/lib/src/lib.rs:10112:50
      |
10112 |                 job_state_matcher: Option<crate::jobs::params::ListJobStateMatcher>,
      |                                                  ^^^^
      |                                                  |
      |                                                  unresolved import
      |                                                  help: a similar path exists: `crate::projects::regions::jobs`

error[E0433]: failed to resolve: unresolved import
     --> gen/dataproc/v1beta2/lib/src/lib.rs:10141:35
      |
10141 |                     value: crate::jobs::params::ListJobStateMatcher,
      |                                   ^^^^
      |                                   |
      |                                   unresolved import
      |                                   help: a similar path exists: `crate::projects::regions::jobs`

error[E0423]: expected value, found struct `locations::LocationsActions`
    --> gen/dataproc/v1beta2/lib/src/lib.rs:3280:13
     |
3280 |             locations::LocationsActions
     |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `locations::LocationsActions { /* fields */ }`?

error[E0423]: expected value, found struct `regions::RegionsActions`
    --> gen/dataproc/v1beta2/lib/src/lib.rs:3284:13
     |
3284 |             regions::RegionsActions
     |             ^^^^^^^^^^^^^^^^^^^^^^^ did you mean `regions::RegionsActions { /* fields */ }`?

error[E0423]: expected value, found struct `autoscaling_policies::AutoscalingPoliciesActions`
    --> gen/dataproc/v1beta2/lib/src/lib.rs:3298:17
     |
3298 |                 autoscaling_policies::AutoscalingPoliciesActions
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `autoscaling_policies::AutoscalingPoliciesActions { /* fields */ }`?

error[E0423]: expected value, found struct `workflow_templates::WorkflowTemplatesActions`
    --> gen/dataproc/v1beta2/lib/src/lib.rs:3302:17
     |
3302 |                 workflow_templates::WorkflowTemplatesActions
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `workflow_templates::WorkflowTemplatesActions { /* fields */ }`?

error[E0423]: expected value, found struct `autoscaling_policies::AutoscalingPoliciesActions`
    --> gen/dataproc/v1beta2/lib/src/lib.rs:6325:17
     |
6325 |                 autoscaling_policies::AutoscalingPoliciesActions
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `autoscaling_policies::AutoscalingPoliciesActions { /* fields */ }`?

error[E0423]: expected value, found struct `clusters::ClustersActions`
    --> gen/dataproc/v1beta2/lib/src/lib.rs:6329:17
     |
6329 |                 clusters::ClustersActions
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `clusters::ClustersActions { /* fields */ }`?

error[E0423]: expected value, found struct `jobs::JobsActions`
    --> gen/dataproc/v1beta2/lib/src/lib.rs:6333:17
     |
6333 |                 jobs::JobsActions
     |                 ^^^^^^^^^^^^^^^^^ did you mean `jobs::JobsActions { /* fields */ }`?

error[E0423]: expected value, found struct `operations::OperationsActions`
    --> gen/dataproc/v1beta2/lib/src/lib.rs:6337:17
     |
6337 |                 operations::OperationsActions
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `operations::OperationsActions { /* fields */ }`?

error[E0423]: expected value, found struct `workflow_templates::WorkflowTemplatesActions`
    --> gen/dataproc/v1beta2/lib/src/lib.rs:6341:17
     |
6341 |                 workflow_templates::WorkflowTemplatesActions
     |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `workflow_templates::WorkflowTemplatesActions { /* fields */ }`?

warning: unreachable expression
    --> gen/dataproc/v1beta2/lib/src/lib.rs:1754:13
     |
1754 | /             Ok(match value {
1755 | |                 _ => {
1756 | |                     return Err(::serde::de::Error::custom(format!(
1757 | |                         "invalid enum for #name: {}",
...    |
1760 | |                 }
1761 | |             })
     | |______________^
     |
     = note: #[warn(unreachable_code)] on by default

warning: unreachable expression
    --> gen/dataproc/v1beta2/lib/src/lib.rs:2397:13
     |
2397 | /             Ok(match value {
2398 | |                 _ => {
2399 | |                     return Err(::serde::de::Error::custom(format!(
2400 | |                         "invalid enum for #name: {}",
...    |
2403 | |                 }
2404 | |             })
     | |______________^

warning: unreachable expression
    --> gen/dataproc/v1beta2/lib/src/lib.rs:9311:25
     |
9311 | /                         Ok(match value {
9312 | |                             _ => {
9313 | |                                 return Err(::serde::de::Error::custom(format!(
9314 | |                                     "invalid enum for #name: {}",
...    |
9317 | |                             }
9318 | |                         })
     | |__________________________^

error: aborting due to 11 previous errors

Some errors have detailed explanations: E0423, E0433.
For more information about an error, try `rustc --explain E0423`.
error: Could not compile `google-dataproc1_beta2`.

To learn more, run the command again with --verbose.
